### PR TITLE
Add ghc-options passthrough

### PR DESCRIPTION
**Pull request by aschmois**
Changes:
- Add optional `ghc-options` passthrough

---

- [X] I manually tested the code locally to make sure that it works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +18,7 @@ import Foo.Handler.GoodbyeWorld
 
 ### Output:
 ```haskell
-{-# LANGUAGE PartialTypeSignatures #-}
+{-# LANGUAGE PartialTypeSignatures -freduction-depth=0 #-}
```
Comment (codygman) [inline README.md]: Can you put a ghc option in a language pragma? No right?

Comment (aschmois) [reply README.md]: Oops, you're right this was just a readme mistake

File: app/ApiModule.hs [hunk 2]
```diff
@@ -12,17 +12,19 @@ import Data.List (sort, (\\))
...
   , imports :: [Module]
   }
 
-renderApiModule :: ApiModule -> Text
-renderApiModule ApiModule { moduleName, imports } =
+renderApiModule :: Config -> ApiModule -> Text
+renderApiModule config ApiModule { moduleName, imports } =
   ( T.unlines
-    $ "{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}"
+    $ "{-# OPTIONS_GHC -fno-warn-partial-type-signatures " <> (ghcOptions config) <> " #-}"
```
Comment (tfausak) [inline app/ApiModule.hs]:
```suggestion
    $ "{-# OPTIONS_GHC -fno-warn-partial-type-signatures " <> ghcOptions config <> " #-}"
```

**Review by tfausak — APPROVED** [review]

**Review by codygman — APPROVED** [review]

**Comment by tfausak** [comment]
Uploaded: https://hackage.haskell.org/package/servant-serf-0.1.1

**Pull request merged**